src/main/resources/default/assets/tycho/scripts/selectableTable.js.pasta

```diff
@@ -43,16 +44,27 @@ sirius.ready(function () {
     function toggleRowSelection(_checkbox) {
         const _row = _checkbox.closest('tr');
         if (_checkbox.checked) {
+            _checkbox.title = "___i18n('selectableTable.row.deselection')";
             _row.classList.add('bg-sirius-blue-faint');
         } else {
+            _checkbox.title = "___i18n('selectableTable.row.selection')";
```

Comment:
**Optional:** Can't we call `toggleSelectAllCheckboxTitle` here instead of duplicating the keys?

Reply:
This is not the title for the `selectAllCheckbox` here. This is the title of an individual checkbox in a row.